js/searchResults.js

```diff
@@ -0,0 +1,91 @@
+class SearchResult {
+  constructor(searchResultElement) {
+    this.searchResultElement = searchResultElement;
+  }
+
+  async getCompanyData(symbol) {
+    try {
+      const url =
+        "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/" +
+        symbol;
+
+      const response = await fetch(url);
+      const result = response.json();
+
+      return result;
+    } catch (error) {
+      return false;
+    }
+  }
+
+  companyInfoPage(name, symbol, image, percentage) {
+    const container = document.createElement("ul");
+    container.classList.add("list-group-flush");
+    container.classList.add("list-group");
+    container.style.borderBottom = "1px solid gray";
+    container.style.width = "fit-content";
+    container.style.alignItems = "center";
+    container.style.padding = "10px";
+    container.style.margin = "auto";
+    container.style.marginTop = "10px";
+
+    const makeLink = document.createElement("a");
+    makeLink.setAttribute("href", "company.html?symbol=" + symbol);
+    makeLink.setAttribute("target", "_blank");
+    makeLink.innerHTML = " " + name + " (" + symbol + ") ";
+    const searchValueRegex = new RegExp(this.searchValue, "gi");
```

Comment:
nice way of implementing milestone 7. I would recommand you to exctract the code that handles the highlighitng of the text into a helper function, and then use it wherever you want to.